libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -404,6 +418,55 @@ static bool _createKeepAliveOperation( const IotMqttNetworkInfo_t * pNetworkInfo
 
 /*-----------------------------------------------------------*/
 
+static IotNetworkError_t _initializeNetworkConnection( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                                       void ** pNetworkConnection,
+                                                       bool * pCreatedNetworkConnection )
+{
+    IOT_FUNCTION_ENTRY( IotNetworkError_t, IOT_NETWORK_SUCCESS );
+    IotNetworkError_t networkStatus = IOT_NETWORK_SUCCESS;
```

Comment:
Why have a separate networkStatus variable?  One should be sufficient.

Reply:
Removed `networkStatus` and used `status` instead.